docs/upgrade-notes/upgrade-task-file-schemas.md

```diff
@@ -0,0 +1,174 @@
+---
+id: update-task-file-schemas
+title: Updates to task granule file schemas
+hide_title: false
+---
+
+## Background
+
+Most Cumulus workflow tasks expect as input a payload of granule(s) which contain the files for each granule. Most tasks also return this same granule structure as output.
+
+However, up to this point, there was inconsistency in the schemas for the granule `files` objects expected by each task. Furthermore, there was no guarantee of consistency between granule `files` objects as stored in the database and the expectations of any given workflow task.
+
+Thus, when performing bulk granule operations which pass granules from the database into a Cumulus workflow, it was possible for there to be schema validation failures depending on which task was used to start the workflow and its particular schema.
+
+In order to rectify this situation, [CUMULUS-2388](https://bugs.earthdata.nasa.gov/browse/CUMULUS-2388) was filed and addressed to create a common granule files schema between nearly all of the Cumulus tasks (exceptions discussed below). The following documentation explains the manual changes you need to make to your deployment in order to be compatible with the updated files schema.
+
+## Updated files schema
+
+The updated granule files schema [can be found here](https://github.com/nasa/cumulus/blob/master/packages/schemas/files.schema.json).
+
+These former properties were deprecated (with notes about how to derive the same information from the updated schema, if possible):
+
+- `filename` - concatenate the `bucket` and `key` values with a directory separator (`/`)
+- `name` - use `fileName` property
+- `etag` - ETags are no longer provided as an individual file property. Instead, a separate `etags` object mapping S3 URIs to ETag values is provided as output from the following workflow tasks (guidance on how to integrate this output with your workflows is provided in the [Upgrading your workflows](#upgrading-your-workflows) section below):
+  - `update-granules-cmr-metadata-file-links`
+  - `hyrax-metadata-updates`
+- `fileStagingDir` - no longer supported
+- `url_path` - no longer supported
+- `duplicate_found` - This property is no longer supported, however `sync-granule` and `move-granules` now produce a separate `granuleDuplicates` object as part of their output. The `granuleDuplicates` object is a map of granules by granule ID which includes the files that encountered duplicates during processing. **Please note** that the `granuleDuplicates` output is purely **informational** and does not have any bearing on the separate configuration for how duplicates should be handled. Guidance on how to integrate `granuleDuplicates` information into your workflow configuration is provided below.
```

Comment:
Consider: 

```suggestion
- `duplicate_found` - This property is no longer supported, however `sync-granule` and `move-granules` now produce a separate `granuleDuplicates` object as part of their output. The `granuleDuplicates` object is a map of granules by granule ID which includes the files that encountered duplicates during processing. Guidance on how to integrate `granuleDuplicates` information into your workflow configuration is provided below.
```

and move 

```
 **Please note** that the `granuleDuplicates` output is purely **informational** and does not have any bearing on the separate configuration for how duplicates should be handled.
 ```
 
 to the details section below.    Low-key feels like that caveat should just be included in the doc description.

Reply:
45a51962d8a34d4ab27944cd5b5e4678112ca7f0